core/src/labels/labels.cpp

```diff
@@ -63,6 +63,32 @@ void Labels::processLabelUpdate(const ViewState& viewState,
     }
 }
 
+
+bool Labels::getLabel(const std::vector<std::unique_ptr<Style>>& _styles,
+                                      const std::vector<std::shared_ptr<Tile>>& _tiles,
+                                      uint32_t _selectionColor, Label*& _label, Tile*& _tile) {
```

Comment:
We would use `**` here, I just feel like pointer indirection is hard to read, but I can change for that.